ceph.spec.in

```diff
@@ -403,7 +402,6 @@ Summary:	OCF-compliant resource agents for Ceph daemons
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
-License:	LGPL-2.0
```

Comment:
> How do we tell what license applies to a given subpackage? I'd like to keep this as low-maintenance as possible.

sorry, no short answer. it depends on what that package is involved with, and it's compile time and runt-time dependencies. for example, if it is a derivative work of GPL licensed software. it's GPL'ed also.

> If we are going to declare a separate/difference license for a subpackage, there would need to be some justification why it is "necessary".

good q. i am just not sure why we have separated licenses for the subpackages in the very beginning. did we add them because it's "necessary"? if not, it'd be great to use a single license.